simple-facebook-instant-articles.php

```diff
@@ -212,9 +212,11 @@ public function gallery_shortcode( $atts, $content = '' ) {
 		ob_start();
 
 		echo '<figure class="op-slideshow">';
```

Comment:
I'm trying to avoid opening/closing PHP tags in non-template files... I know Eprhaim isn't a fan.

However closing the PHP tags won't please codesniffer.  Specifically its looking for variables that might contain untrusted/insecure data that is being output. This is OK because its just a hardcoded string and not a security concern. No need to escape.